ui/src/types/dygraphs.ts

```diff
@@ -826,12 +826,13 @@ export declare class DygraphClass {
   public visibility(): boolean[]
 
   /**
-   * Changes the visiblity of a series.
+   * Changes the visibility of a series.
    *
    * @param {number} num the series index
    * @param {boolean} value true or false, identifying the visibility.
    */
-  public setVisibility(num: number, value: boolean): void
+
+  public setVisibility(visibilities: number | boolean[], value?: boolean): void
```

Comment:
Is there a better way to define this? It can be used to change the visibility of a single series via `index, value` but can also be passed an array of booleans that maps the entire set of visibilities